### PR TITLE
Fixed style of participant badge

### PR DESCRIPTION
**Pull request by priyanshu-kun**
close issue: https://github.com/jitsi/jitsi-meet/issues/12878

This occurs when the screen width and height become smaller than 368 x 619 or badge number become two or more digits, causing the badge value to overflow the badge container. To address this issue, I made the badge more responsive for larger numbers and smaller devices.

Before: 
Screenshot from 2023-02-07 23-05-17

After fix:
Screenshot from 2023-02-08 00-13-11

### CODE REVIEW

**Comment by priyanshu-kun** [comment]
Done, I have signed that.

Individual Name: Priyanshu Sharma
Mailing Address: priyanshusharma507@protonmail.com

**Comment by saghul** [comment]
@robertpin can you PTAL?

**Comment by priyanshu-kun** [comment]
> @robertpin can you PTAL?

Sorry, I don't understand can you be more elaborate!

**Comment by priyanshu-kun** [comment]
@robertpin

File: react/features/participants-pane/components/web/ParticipantsCounter.tsx [hunk 1]
```diff
@@ -9,15 +9,18 @@ const useStyles = makeStyles()(theme => {
     return {
         badge: {
             backgroundColor: theme.palette.ui03,
-            borderRadius: '100%',
+            borderRadius: '12px',
```
Comment (robertpin) [inline react/features/participants-pane/components/web/ParticipantsCounter.tsx:12]: This change isn't necessary

File: react/features/participants-pane/components/web/ParticipantsCounter.tsx [hunk 2]
```diff
@@ -9,15 +9,18 @@ const useStyles = makeStyles()(theme => {
...
             height: '16px',
-            width: '16px',
+            minWidth: '16px',
             color: theme.palette.text01,
             ...withPixelLineHeight(theme.typography.labelBold),
+	    lineHeight: '20px',
```
Comment (robertpin) [inline react/features/participants-pane/components/web/ParticipantsCounter.tsx:17]: Please remove this. The line-height is part of the typography (set in the line above)

File: react/features/participants-pane/components/web/ParticipantsCounter.tsx [hunk 3]
```diff
@@ -9,15 +9,18 @@ const useStyles = makeStyles()(theme => {
...
             pointerEvents: 'none',
             position: 'absolute',
             right: '-4px',
-            top: '-3px'
+            top: '-3px',
+	    textAlign: 'center',
```
Comment (robertpin) [inline react/features/participants-pane/components/web/ParticipantsCounter.tsx:22]: Please indent at the same level as the line above (the line below should also be indented correctly)

**Review by robertpin — CHANGES_REQUESTED** [review]

**Comment by priyanshu-kun** [comment]
@robertpin done.

**Review by robertpin — APPROVED** [review]

**Pull request merged**